### PR TITLE
chore : Uploade SQL homework

### PR DESCRIPTION
**Pull request by aimanzi**

### CODE REVIEW

File: SQL/booktable.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+create table Books (
+Book_Name varchar(20),
+Auther varchar(15),
+Book_language varchar (10),
+Pages Number numeric
```
Comment (EliasSalom) [inline SQL/booktable.sql:5]: you need to choose between the number or numeric!!

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please check my comment

File: SQL/booktable.sql (hunk 1 above)
Comment (aimanzi) [reply SQL/booktable.sql]: problem solved

**Review by EliasSalom — APPROVED** [review]
LGTM

**Comment by EliasSalom** [comment]
there is some conflicts that must be resolved

**Comment by EliasSalom** [comment]
**don't close any pull request without comment, this is not allowed**

**Pull request closed without merging**